docs/android/v2/advanced-features/simulcast.mdx

```diff
@@ -0,0 +1,46 @@
+---
+ title: Simulcast
+ nav: 8.04
+---
+
+## Minimum Requirements
+
+- SDK version 2.5.2 or higher
+- Simulcast enabled in room template
+
+## 100ms SDK Simulcast APIs
+
+You interact with the simulcast feature by setting `HMSRemoteVideoTrack`'s `setLayer` function. Let's look at the track interface:
+
+```kotlin
+class HMSRemoteVideoTrack {
+	
+/**
+   * Public api to set layer to [HMSLayer.HIGH], [HMSLayer.LOW], [HMSLayer.MEDIUM]
+   */
+  fun setLayer(HMSLayer: HMSLayer)
+		
+	/**
+   * Public API to get layers definition
+   */
+  fun getLayerDefinition() : List<HMSSimulcastLayerDefinition>
+
+	/**
+   * Public API to get current layer value
```

Comment:
`Public API to get current layer value` to `Public API to get the current selected HMSLayer`